neoctl.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+
+# Copyright (c) 2002-2015 "Neo Technology,"
+# Network Engine for Objects in Lund AB [http://neotechnology.com]
+#
+# This file is part of Neo4j.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from sys import argv, exit
+from os import name
+from getopt import getopt
+from subprocess import call, Popen, PIPE
+
+
+is_windows = (name == 'nt')
+neo4j_home = '.'
+
+
+def main():
+    global neo4j_home
+    if len(argv) <= 1:
+        print_help()
+        exit()
+    try:
+        opts, args = getopt(argv[1:], "h", ["start=", "stop="])
+    except getopt.GetoptError as err:
+        print str(err)
```

Comment:
Thank for all your catch for python 3. I was wondering why Nigel always uses stdout instead of print.